scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2008. All rights reserved.
+#
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+CHAIN=RH-Firewall-1-INPUT
+OP=$1
+rule="RH-Firewall-1-INPUT -p tcp -m conntrack --ctstate NEW -m tcp --dport 6640 -j ACCEPT"
+
+if [[ "${OP}" == "start" ]]; then
+    if iptables -S | grep -e "${rule}" > /dev/null
+    then
+        echo "port 6640 is opened already"
+        exit
+    else
+        iptables -A $rule
+        service iptables save
+    fi
+elif [[ "${OP}" == "stop" ]]; then
+    iptables -D $rule
+    service iptables save
+else
+    echo $"Usage: $0 {start|stop}"
+    exit
```

Comment:
$"Usage: $0 {start|stop}" 
will this clearly show the error happens or why that occurs?